src/Compilers/Core/Portable/InternalUtilities/EncodingExtensions.cs

```diff
@@ -28,7 +28,9 @@ internal static int GetMaxCharCountOrThrowIfHuge(this Encoding encoding, Stream
                 return maxCharCount;
             }
 
-#if WORKSPACE
+#if CODE_STYLE
```

Comment:
@dotnet/roslyn-compiler The only changes to compiler layer are in 3 files: this one and the 2 that follow. The changes are just adding a new `#if CODE_STYLE` for utility methods that are now also included in the CodeStyle layer.